docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md

```diff
@@ -0,0 +1,725 @@
+installation onto the VM.
+
+Wait for the Edge Installer to complete copying content to the VM, which may take a few minutes. The VM will reboot by
+default upon completion.
+
+When you see the image below, right-click the VM, select **Stop**, and then click **Power Off** to turn it off before
+the reboot starts.
+
+![A screenshot of the VirtualBox VM after installation.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_vm-reboot.webp)
+
+Next, click **Settings** and select **Storage**.
+
+Select the Edge Installer ISO and click **Remove Attachment** to remove it from your VM. Confirm the deletion with
+**Remove** and click **OK** to close the settings window.
+
+![A screenshot of the VirtualBox VM storage configuration.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_vm-remove-iso.webp)
+
+### Deploy Edge Host
+
+Select the VirtualBox VM you created and click **Start** to turn it on. The VM will boot and get an IP address from the
+bridged network of the host machine. This address should be on the same subnet as the host machine.
+
+After a few minutes, the VM screen displays an IP address and registers automatically in Palette as an Edge host using
+the provided Palette registration token.
+
+![A screenshot of the Edge host.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_edge-host.webp)
+
+### Validate the Edge Host Registration
+
+Log in to [Palette](https://console.spectrocloud.com/) and select **Clusters** from the left **Main Menu**. Click on the
+**Edge Hosts** tab to view the registered hosts.
+
+Confirm your Edge host is listed as **Healthy** and with a **Ready** status. The **Machine ID** should match the ID
+displayed on your VM's screen.
+
+![A screenshot of the Edge host in Palette.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_edge-host-palette.webp)
+
+## Deploy Edge Cluster
+
+The following steps will guide you through deploying the Edge cluster infrastructure.
+
+Select **Clusters** from Palette's left **Main Menu** and click on **Create Cluster**. Ensure you are in the **Default**
+project.
+
+Palette will prompt you to select the type of cluster. Select **Edge Native** and click the **Start Edge Native
+Configuration** button.
+
+In the **Basic Information** section, assign the name **edge-vbox-cluster**, a brief cluster description, and assign the
+tag **env:edge**. Click **Next**.
+
+Select the cluster profile you created in the [Create Cluster Profile](#create-cluster-profile) section of this tutorial
+and click **Confirm**.
+
+The **Cluster Profile** section displays all the layers in the cluster profile. Click on **Next** to proceed.
+
+![A screenshot of the cluster profile during cluster deployment.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_cluster-deployment-profile.webp)
+
+In the **Cluster Config** section, provide a Virtual IP (VIP) address for the Edge cluster. This address must be an
```

Comment:
In the context of virtual box, maybe a little help in identifying the network that the VM is on?

Reply:
huum. The VM will be in the same network as the host machine, as mentioned in the `Deploy VirtualBox VM` section. I added a tip to help the user scan the network and check the used IP addresses.